addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js

```diff
@@ -49,13 +47,6 @@ const PublicLivechatWindow = Widget.extend({
     async start() {
         this.$input = this.$('.o_composer_text_field');
         this.$header = this.$('.o_thread_window_header');
-        const options = {
-            displayMarkAsRead: false,
-        };
-        if (this.messaging.publicLivechatGlobal.publicLivechat.legacyPublicLivechat._type === 'document_thread') {
-            options.displayDocumentLinks = false;
-        }
```

Comment:
this condition has completely disappeared from the new code, is this intentional?

Reply:
Yes, it was moved to model, and it's been simplified to just passing `{ displayMarkAsRead: false }` as option, because the code of public livechat don't deal with `document_thread` at all.